tests/lib/rules/force-types-on-object-props.js

```diff
@@ -0,0 +1,320 @@
+/**
+ * @author Przemysław Jan Beigert
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/force-types-on-object-props')
+
+const templateExport = (prop) => `
+export default {
+  props: ${prop}
+}
+`
+
+const templateSetup = (prop) => `
+<script setup lang="ts">
+defineProps(
+  ${prop}
+)
+</script>
+`
+
+const templateExtend = (prop) => `
+export default Vue.extend({
+  props: 
+    ${prop}
+});
```

Comment:
I see that reusing these functions saves a lot of duplicated lines, but I think the test cases would be much clearer if those strings would be inlined.